src/web/mjs/engine/I18n.mjs

```diff
@@ -0,0 +1,38 @@
+export default class I18n {
```

Comment:
Class wasn't working: rewrote it (you could have used a node lib)
`export default class I18n {
    constructor() {

        this.lang = localStorage.getItem("lang") ?? "en";
        this.dic = undefined;
        this.getDictionary(`lib/hakuneko/lang/${this.lang}.json`);
    } 

    async getDictionary(url) {
        try {
            console.log('Loading translation dictionary: ',url);
            const langfile = await fetch(url);
            const tr = await langfile.json();
            this.dic = tr;
            return tr;
        } catch (e) {
            console.error('Unable to load dictionary', url);
        }
    }

    translate(key, def = '') {
        var tr = key.split('.').reduce(function(obj, key) {
            return obj && obj[key];
        }, this.dic);
        return tr ?? def;
    }
}`